### ISSUE TITLE
Creating new indexes doesn't assign

### ISSUE THREAD

**Issue by wockan** [datarank/tempest#15]
Hi

We have a problem with tempest and ES 1.7.3.
When we add a new index it stays unassigned. 
Also this is our first time using this plugin.

In the log we can see an exception:

```
[2015-10-19 11:39:16,505][INFO ][cluster.metadata         ] [es0] [index_1] deleting index
[2015-10-19 11:39:16,598][INFO ][com.datarank.tempest.allocator.LocalMinimumShardsAllocator] [es0] Rebalancing.
[2015-10-19 11:39:16,602][INFO ][com.datarank.tempest.allocator.LocalMinimumShardsAllocator] [es0] [0] unassigned, [0] ignored unassigned.
[2015-10-19 11:39:16,602][INFO ][com.datarank.tempest.allocator.LocalMinimumShardsAllocator] [es0] Already balanced to 1.1517687735640945, exiting rebalance.
[2015-10-19 11:39:17,796][INFO ][cluster.metadata         ] [es0] [index_1] creating index, cause [api], templates [template_1], shards [5]/[0], mappings [
.... A LOT OF MAPPINGS....
]
[2015-10-19 11:39:17,800][INFO ][com.datarank.tempest.allocator.LocalMinimumShardsAllocator] [es0] Allocating unassigned shards.
[2015-10-19 11:39:17,806][WARN ][com.datarank.tempest.allocator.LocalMinimumShardsAllocator] [es0] [Ljava.lang.StackTraceElement;@7c49f4b9
[2015-10-19 11:39:17,886][INFO ][com.datarank.tempest.allocator.LocalMinimumShardsAllocator] [es0] Rebalancing.
[2015-10-19 11:39:17,886][INFO ][com.datarank.tempest.allocator.LocalMinimumShardsAllocator] [es0] 5 unassigned shards and 0 ignored unassigned shards remain, exiting rebalance.
```

We then have to manually assign the index using:
```
curl -XPOST 'localhost:9200/_cluster/reroute' -d '{
    "commands": [{
        "allocate": {
            "index": "index_1",  
            "shard": x,           // 0 1 2 3 4  
            "node": "es0",          
            "allow_primary": 1
        }
    }]
}'
```

Are we missing a setting or something?

**Comment by thekad** [comment]
I don't think this is specific to 1.7.3, but rather a bug in tempest 1.1.0. I am running tempest in a test cluster with ES 1.4.2 and I can see the problem happen there as well.

**Comment by aewhite** [comment]
This is resolved in Tempest 2.0.2
